src/sentry/api/endpoints/organization_profiling_profiles.py

```diff
@@ -96,7 +97,13 @@ def get(self, request: Request, organization: Organization) -> HttpResponse:
                 request.query_params.get("query", None),
             )
         else:
-            profile_ids = get_profile_ids(params, request.query_params.get("query", None))
+            if "fingerprint" in request.query_params:
+                function_fingerprint = request.query_params["fingerprint"]
```

Comment:
I'm not sure if you need to cast the fingerprint as an int here prior to using it in the query. It may be possible that if you pass it as a string, it'll end up creating a condition where the fingerprint (which is an int) is compared to a string and cause a type mismatch error.

Reply:
It's not really needed. I've tested this E2E locally and it works as expected.

I think it's still a good point though.

I'll add the explicit cast regardless so we don't leave any space for future doubts/bugs. 👍🏻